updates/33.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE person ADD COLUMN GND varchar(255) DEFAULT NULL;
```

Comment:
When adding columns, please don't just add them to the end of the table. try to add them where similar fields are. In this case i suggest you add this to the statement:
`AFTER PKZ`

Reply:
done